docs/releases/1.9.2.txt

```diff
@@ -71,3 +71,6 @@ Bugfixes
 
 * Fixed a crash when using a reverse ``OneToOneField`` in
   ``ModelAdmin.readonly_fields`` (:ticket:`26060`).
+
+* Fixed a crash when using the ``migrate`` command with the ``--run-syncdb``
+  option (:ticket:`26135`).
```

Comment:
The failure requires a really specific of be triggered.  do you think the following is too technical?

> Fixed a crash when calling the ``migrate`` command in a test case with the ``available_apps`` setting pointing to an applications with disabled migrations using the ``MIGRATION_MODULES`` setting.